modules/installation-azure-limits.adoc

```diff
@@ -184,6 +184,17 @@ ifndef::ash[]
 endif::ash[]
 |The internal load balancer, each of the three control plane machines, and each
 of the three worker machines each use a private IP address.
+
+ifndef::ash[]
+|Spot VM vCPUs
+|Number of compute nodes*2
```

Comment:
I'm not sure this information belongs here. A default value should be a fixed value. Right? Or have I missed something?

Reply:
@ctauchen But, that's the minimum required value and it varies depending on the number of compute nodes in a cluster. Since we are adding at least twice the number in the description, we can't leave that field empty or add a zero, can we?

Reply:
Hmm. This does get confusing. Column 2 describes the minimum Azure resources you need to perform a default OSP installation.

For spot VMs, that minimum = 0, because you don't need to use spot VMs at all. All the other components in this table are required.

My worry is that someone who has no need of spot VMs would think they need to increase the availability of this service, at some cost, just to perform the default installation.

@codyhoag @kalexand-rh You two worked with this table before. Do you have any thoughts here?

Reply:
@codyhoag Thanks for your input @ctauchen I have incorporated Cody's suggestion.

Reply:
For what it's worth, I think the extra clarification of Cody's Col 2 text is helpful. But it's your call. 

LGTM

Reply:
comment addressed.